apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useGetMembers.ts

```diff
@@ -0,0 +1,44 @@
+import { memberCredentialsSelector, trustchainSelector } from "@ledgerhq/trustchain/store";
+import { useSelector } from "react-redux";
+import { useTrustchainSdk } from "./useTrustchainSdk";
+import { useQuery } from "@tanstack/react-query";
+import { QueryKey } from "./type.hooks";
+import { useTranslation } from "react-i18next";
+import { createCustomErrorClass } from "@ledgerhq/errors";
+
+export const TrustchainNotFound = createCustomErrorClass("TrustchainNotFound");
+export const MemberCredentialsNotFound = createCustomErrorClass("MemberCredentialsNotFound");
+
+export function useGetMembers() {
+  const sdk = useTrustchainSdk();
+  const trustchain = useSelector(trustchainSelector);
+  const memberCredentials = useSelector(memberCredentialsSelector);
+  const { t } = useTranslation();
+
+  function getMembers() {
+    if (!memberCredentials) {
+      throw new MemberCredentialsNotFound(
+        t("walletSync.walletSyncActivated.errors.memberCredentials"),
```

Comment:
It's a bit an antipattern with the way we word error here: you can just throw new FooError() and in the wording file you just need to declare a wording in the errors object, it will be mapped automatically by the error name

Reply:
Ok !